lib/room.rb

```diff
@@ -1,12 +1,24 @@
-class Rom
+class Room
+
+  attr_reader :length, :width, :category
```

Comment:
Since we never need to call `length` or `width` as methods, they do not need to be attr_readers